app/helpers/form_helper.rb

```diff
@@ -442,6 +442,15 @@ def form_select_f(f, attr, array, select_options = {}, html_options = {})
     include_blank = select_options.delete(:include_blank)
     if include_blank
       addClass html_options, "include_blank"
+      blank_value = include_blank.is_a?(TrueClass) ? nil : include_blank
```

Comment:
I'd opt for `include_blank == true`. Same thing, expressed differently

Reply:
include_blank is a string or a boolean

Reply:
I meant, `include_blank == true ? nil : include_blank` instead of checking if it is an instance of TrueClass

Reply:
I see, sure